index.html

```diff
@@ -59,5 +59,163 @@ <h3>LET'S CONNECT</h3>
         </ul>
       </section>
     </main>
+    <!-- Cards section -->
+    <section class="main-container-projects">
+      <div class="cards card-1">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio.png" alt="person" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
+            <span>CANOPY</span>
+            <input type="radio" checked />Back End Dev
+            <input type="radio" checked />2015
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="skills">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+          <button class="see-project">See Project</button>
+        </article>
+      </div>
+
+      <div class="cards card-2">
+        <div class="main-img">
+          <img src="./img/Snapshoot Portfolio (1).png" alt="calendar" />
+        </div>
+        <article class="second-container">
+          <h2>Tonic</h2>
+          <div class="horizontal-section">
+            <span>CANOPY</span>
+            <input type="radio" checked />Back End Dev
+            <input type="radio" checked />2015
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="skills">
+            <div>html</div>
+            <div>css</div>
+            <div>javascript</div>
+          </div>
+          <button class="see-project">See Project</button>
+        </article>
+      </div>
```

Comment:
- [ ] Please note that the minimum mobile screen size should be `375`px so your mobile design should be responsive from that size until the desktop screen size but from the video below, you can notice that from `390`px it wasn't responsive all through to `482` px before it began to look okay. So kindly ensure that from your design is responsive from '375' px and above. :+1: 
- Video Reference
[Screencast from 02-17-2023 05:04:26 AM.webm](https://user-images.githubusercontent.com/102699958/219554565-a9010c2f-bd9a-47cc-b35d-13ece97553a3.webm)